### PR TITLE
Updates to livestream consulting page

### PR DESCRIPTION
**Pull request by tiagov8**
Changes from @apforsyth via **Re: 🖥️ SSW Livestreaming Services – Web page  update #1 of #2**

### CODE REVIEW

**Review by amankumarrr — APPROVED** [review]

File: content/video-production/live-events.mdx [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 seo:
-  title: Live Events
+  title: 'Filming, Livestreaming, Recording & Video Editing'
   description: >-
     SSW Consulting has over 30 years of Microsoft software and web development
```
Comment (camillars) [inline content/video-production/live-events.mdx]: 1. Update the SEO description to:
Our live streaming services feature top-notch multi-camera setups to ensure a flawless audience experience.

File: content/video-production/live-events.mdx [hunk 2]
```diff
@@ -43,46 +43,25 @@ Using the latest technologies and with coaching from our media team SSW can crea
...
-An indicative price for livestreaming an event like this would be approximately $3,000 AUD + GST per day.
+We, at SSW, utilize this hall for many of our corporate training events and we are now making it available to other businesses looking for an uncomplicated venue to hold their small events without that hassle of negotiating with large conference hall coordinators. Learn more at [SSW Chapel website](https://www.ssw.com.au/ssw/Events/Venues/Hire-Conference-Room.aspx).
 
-## Install Livestream System
 
-### Description
+An indicative price for livestreaming an event like this would be approximately $9,000 AUD + GST per day, including preproduction and  livestreaming with a dedicated Event Master and a crew of 2 SSW TV team members.
 
-SSW is one of the first software consultancies in Australia to regularly Live Stream high quality presentations. We have recognised that other businesses have a need to live stream. If you are looking at live streaming your events, then our technicians can install a system for you and train your staff to use it.
+Please call for a quote to meet your specific requirements.
```
Comment (camillars) [inline content/video-production/live-events.mdx]: 2. change to
Please call for a quote that meets your prerequisites.

**Review by camillars — CHANGES_REQUESTED** [review]

**Comment by camillars** [comment]
@bradystroud 
1. please track SEO changes at https://sswcom.sharepoint.com/:x:/r/sites/SSWMarketing/Shared%20Documents/General/SEO/On-Page-SEO-Plan-Australia.xlsx

File: content/video-production/live-events.mdx (hunk 1 above)
Comment (tiagov8) [reply content/video-production/live-events.mdx]: Done

File: content/video-production/live-events.mdx (hunk 2 above)
Comment (tiagov8) [reply content/video-production/live-events.mdx]: Done

**Comment by tiagov8** [comment]
Hi @bradystroud 

I've done the changes requested by @camillars 

Looking at the page this morning I found this broken image (I assume the placeholder for videos).

- [x] 1. Create a new Issue for this and link from here if you find necessary
- [x] 2. Please fix this before/while deploying my changes 

Screenshot 2023-07-12 at 9 56 53 AM

Screenshot 2023-07-12 at 9 58 50 AM

Cheers,

**Comment by bradystroud** [comment]
Thanks @tiagov8 

>  1. Create a new Issue for this and link from here if you find necessary

#1016

**Review by camillars — APPROVED** [review]

**Comment by bradystroud** [comment]
@camillars 

> 1. please track SEO changes at [https://sswcom.sharepoint.com/❌/r/sites/SSWMarketing/Shared%20Documents/General/SEO/On-Page-SEO-Plan-Australia.xlsx](https://sswcom.sharepoint.com/:x:/r/sites/SSWMarketing/Shared%20Documents/General/SEO/On-Page-SEO-Plan-Australia.xlsx)

Done - I dont like this file but still haven't come up with a better solution...

image

**Comment by camillars** [comment]
Was the previous info tracked too?











Camilla Rosa Silva   |   SSW Marketing Manager<http://www.ssw.com.au/People/Camilla-Rosa%20Silva?utm_source=emailsignature>   |    +61 426 007 665<https://www.ssw.com.au/ssw/Company/Offices/?utm_source=emailsignature>   |   Ping me on ***@***.***>

**Pull request merged**